src/pages/RepoPage/FailedTestsTab/FailedTestsPage/hooks/useInfiniteTestResults/useInfiniteTestResults.tsx

```diff
@@ -181,7 +183,8 @@ interface UseTestResultsArgs {
     testResults: TestResult[]
     pageInfo: { endCursor: string | null; hasNextPage: boolean }
     private: boolean | null
-    plan: PlanName | null
+    planName: PlanName | null
+    isFreePlan: boolean | null
```

Comment:
because it says | null here

Reply:
guh good catch, I'll update. It shouldn't be nullable iirc